CHANGELOG.md

```diff
@@ -25,7 +25,12 @@ the detailed section referring to by linking pull requests or issues.
 #### Removed
 
 #### Fixed
+
 - Placeholder URLs missing "/control/" path
+- Fixed Keyword select not adding keywords on input field blur, causing loss of
+  input.
+- Fixed transfer dialog submitting on cancel
+- Fixed transfer dialog validation not working
 
```

Comment:
Introduced reusability of Data Address Form?

Reply:
I'd say that's too technical to be relevant. I'd rather focus on "product related" changes we got done. Despite producing code, the above refactoring was very natural

Reply:
Oh and we did not introduce reusability of the data address form